pkg/component/vpnshoot/testdata/monitoring_alertingrules.yaml

```
@@ -23,6 +23,25 @@ tests:
         description: vpn-shoot deployment in Shoot cluster has 0 available pods. VPN won't work.
         summary: VPN Shoot deployment no pods
 
+- interval: 30s
+  input_series:
+  # VPNHAShootNoPods
+  - series: 'kube_statefulset_status_replicas_ready{statefulset="vpn-shoot"}'
+    values: '0+0x60'
+  alert_rule_test:
+  - eval_time: 30m
+    alertname: VPNHAShootNoPods1
```

Is there a reason for the `1` at the end of the name? With the additional `HA` in the name the alert names are already different.

```suggestion
    alertname: VPNHAShootNoPods
```

---

fixed 924ce09abe11dd44b7b69bc4aa0e19611dbba395